src/itest/kotlin/nl/lifely/zac/itest/ZakenRESTServiceTest.kt

```diff
@@ -105,18 +105,12 @@ class ZakenRESTServiceTest : BehaviorSpec({
                     "  \"betrokkeneIdentificatie\": \"$TEST_PERSON_HENDRIKA_JANSE_BSN\"\n" +
                     "}"
             )
-            Then("the response should be a 400 bad request HTTP response") {
-                response.code shouldBe HTTP_STATUS_BAD_REQUEST
```

Comment:
This constant's import should be removed

Reply:
Cahnged this to expect HTTP_STATUS_OK; since that is the solution

Reply:
Replaced it by the OK 200 that is expected here